src/Components/Board.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import {SquareProps} from '../Types/TypeProps'
+import Square from "./Square";
+
+/**
+ * Handle board's cells display logic
+ * @param props 
+ * @returns 
+ */
+const Board: React.FC<SquareProps> = (props) => {
```

Comment:
I would 
```suggestion
const Board = ({ value, isWinnerCell,  squares, position, extendHandleClick, handleClick }: SquareProps) => {
```

For React components, I'd let TypeScript implicitly determine the return type. I'd also always destructure props here, you'll save a lot of `props.`